test/shaderdb/gfx9/PipelineVsFs_TestRelocatableInOutMapping.pipe

```diff
@@ -0,0 +1,114 @@
+; This test checks that the mapping for the outputs of the vertex shader
+; match the mapping for the inputs of the fragment shader.
+
+; BEGIN_SHADERTEST
+; RUN: amdllpc -use-relocatable-shader-elf -auto-layout-desc -spvgen-dir=%spvgendir% -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST: (VS) Output: loc = 0  =>  Mapped = [[loc0:[0-9]+]]
+; SHADERTEST: (VS) Output: loc = 1  =>  Mapped = [[loc1:[0-9]+]]
+; SHADERTEST: (VS) Output: loc = 3  =>  Mapped = [[loc3:[0-9]+]]
+; SHADERTEST: (VS) Output: loc = 4  =>  Mapped = [[loc4:[0-9]+]]
+; SHADERTEST: (FS) Input: loc = 0  =>  Mapped = [[loc0]]
+; SHADERTEST: (FS) Input: loc = 1  =>  Mapped = [[loc1]]
+; SHADERTEST: (FS) Input: loc = 3  =>  Mapped = [[loc3]]
+; SHADERTEST: (FS) Input: loc = 4  =>  Mapped = [[loc4]]
```

Comment:
You could strengthen this test by checking that the vertex shader's export instructions look right. Also, maybe a test where the pixel shader only loads one (non-loc-0) input, and check that the v_interps are right.